lnwallet/channel.go

```diff
@@ -2590,8 +2605,7 @@ func (lc *LightningChannel) CreateCloseProposal(feeRate uint64) ([]byte, error)
 //
 // NOTE: The passed local and remote sigs are expected to be fully complete
 // signatures including the proper sighash byte.
-func (lc *LightningChannel) CompleteCooperativeClose(localSig,
-	remoteSig []byte, proposedFee uint64) (*wire.MsgTx, error) {
+func (lc *LightningChannel) CompleteCooperativeClose(localSig, remoteSig []byte) (*wire.MsgTx, error) {
```

Comment:
Fee rate should still be passed in, otherwise this will sign a commitment transaction with the prior fee of the latest commitment state.